.gitignore

```diff
@@ -0,0 +1 @@
+*.test
```

Comment:
Happy to open something separate for this, but it doesn't seem like there's any .gitignore in place at the moment